### PR TITLE
AWS SDK Version Upgrade to v1.44.3

### PR DESCRIPTION
**Pull request by Jamil-Najafov**
Hi! AWS SDK has a number of stability upgrades that'd be great to have.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,12 +1,10 @@
 module github.com/PaddleHQ/go-aws-ssm
 
-go 1.12
+go 1.17
```
Comment (geototti21) [inline go.mod:3]: This change will force everyone that uses the library to bump to 1.17 Go version, is it possible to still keep 1.12 here?

**Review by geototti21 — CHANGES_REQUESTED** [review]
Thanks for the changes @Jamil-Najafov 🙌 

Can you please update travis file, seems like the `tip` go is failing, maybe we can lock it down to 1.17 or 1.18 instead of tip?

File: go.mod (hunk 1 above)
Comment (Jamil-Najafov) [reply go.mod:3]: hey, sure I'll check v1.12 compatibility.

**Comment by NuclearError** [comment]
Guys, I probably shouldn't still have access to this repo. Please remove me and look into tightening up the security around this.

**Comment by geototti21** [comment]
> Guys, I probably shouldn't still have access to this repo. Please remove me and look into tightening up the security around this.

Hey @NuclearError , thanks for flagging but this is a public repo under Apache licence, I think you are getting notifications because you follow the repo since when you working for Paddle 🙂

**Comment by geototti21** [comment]
we upgraded dependencies on #37

**Pull request closed without merging**